runtime/src/accounts_db.rs

```diff
@@ -6807,12 +6821,6 @@ impl AccountsDb {
         Ok((accounts_hash, total_lamports))
     }
 
-    pub fn get_accounts_hash(&self, slot: Slot) -> AccountsHash {
-        let bank_hashes = self.bank_hashes.read().unwrap();
-        let bank_hash_info = bank_hashes.get(&slot).unwrap();
-        bank_hash_info.accounts_hash
```

Comment:
so these were the unwraps which previously existed and would have panic'd on none.